### DISCUSSION TITLE
How to reset 2fa for a user

### DISCUSSION THREAD

**Discussion by AlexBrown-at** [community/community#30119] [Other Feature Feedback, Questions, & Ideas] [answered]
I have two users that switched phones and they can no longer log into github due to duo 2fa, 

Is there a way to "reset" 2fa on their github account 

Removing and re-adding them didnt work

**Comment by airtower-luna** [comment, marked as answer]
Check [Recovering your account if you lose your 2FA credentials](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials) for the options they have. Recovery codes are the best option.
